### PR TITLE
added a style guide

### PR DESCRIPTION
**Pull request by abrahammurciano**

### CODE REVIEW

File: style_guide.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Please follow the guidelines stated here when contributing to this repository.
+
+## Spacing
+
+- Always use tabs. Spaces suck.
+- Avoid trailing spaces at the end of lines.
+- Avoid trailing empty lines at the end of files.
+- Surround operators with a single space (except for when using `=` in parameter lists).
+- Leave two empty lines between functions.
+- Optionally leave one empty line between logical blocks of code in the same function.
```
Comment (probably-raptors) [inline style_guide.md:12]: If it's optional why mention it? It should either be enforced or not mentioned (I think enforced)

File: style_guide.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## Naming
```
Comment (probably-raptors) [inline style_guide.md:14]: Should also mention our preferred prefixes and names for files/directories

File: README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ Bot for JCT ESP CompSci Discord server
 
 ## Contributors
 
-This bot was created by the contribution of the following members. If you would like to contribute, please be in touch with one of the current contributors and then take a look at [the contributing guide](https://github.com/DenverCoder1/jct-discord-bot/blob/main/Contibuting.md)
+This bot was created by the contribution of the following members. If you would like to contribute, please be in touch with one of the current contributors and then take a look at [the contributing guide](https://github.com/DenverCoder1/jct-discord-bot/blob/main/contibuting.md)
```
Comment (DenverCoder1) [inline README.md:7]: Link is spelled wrong.

Should be able to replace with a relative link

Ex. `[Link](contributing.md)`

File: contributing.md [hunk 4]
```diff
@@ -24,7 +24,7 @@ This name will be referred to as _myfeature_ throughout this guide.
 
 Now you can write your code to implement your feature. You can commit and push to the dedicated branch as often and whenever you like.
 
-Always use tabs. Spaces suck.
+Please check out the [style guide](https://github.com/DenverCoder1/jct-discord-bot/blob/main/Contributing.md).
```
Comment (DenverCoder1) [inline contributing.md:27]: Wrong link

File: style_guide.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+## Spacing
+
+- Always use tabs. Spaces suck.
+- Avoid trailing spaces at the end of lines.
+- Avoid trailing empty lines at the end of files.
+- Surround operators with a single space (except for when using `=` in parameter lists).
+- Leave two empty lines between functions.
+- Optionally leave one empty line between logical blocks of code in the same function.
+
```
Comment (DenverCoder1) [inline style_guide.md:13]: Suggest installing Prettier extension?

I just let them decide how to format the code lol

**Review by DenverCoder1 — CHANGES_REQUESTED** [review]

File: style_guide.md (hunk 5 above)
Comment (abrahammurciano) [reply style_guide.md:13]: I think they use spaces... will check out prettier, tho I currently use the Python extension for formatting which uses autopep8

File: style_guide.md (hunk 1 above)
Comment (abrahammurciano) [reply style_guide.md:12]: Logical blocks is kind of a blurry definition. This is kind of a suggestion for the devs to be aware that it should be done. Regardless, I'll remove "optionally".

File: style_guide.md (hunk 5 above)
Comment (DenverCoder1) [reply style_guide.md:13]: IMO doesn't matter as long as its readable. Spaces are fine if it's consistent.

File: style_guide.md (hunk 2 above)
Comment (abrahammurciano) [reply style_guide.md:14]: Such as? Stuff like "name you cog class file `my_feature_cog.py`" has been changed. Check out #28

File: style_guide.md (hunk 5 above)
Comment (abrahammurciano) [reply style_guide.md:13]: Actually just checked, prettier doesn't have Python support. They dont reference a python plugin in their docs either. We may as well use autopep8 which is the official one. I dont think it converts tabs or spaces tho

Comment (abrahammurciano) [reply style_guide.md:14]: We're using Black (but with tabs)

**Review by DenverCoder1 — APPROVED** [review]

**Pull request merged**